cpp/lib/answer/json.cpp

```diff
@@ -28,6 +28,7 @@ help *
 #include "../common/macros.hpp"
 #include "../common/native.hpp"
 #include "../common/utils.hpp"
+#include <sstream>
```

Comment:
I've followed a convention where I've put all inbuilt standard headers above my local headers. Can you move this one to the top as well?